Framework/Algorithms/test/MayersSampleCorrectionStrategyTest.h

```diff
@@ -80,22 +87,29 @@ class MayersSampleCorrectionStrategyTest : public CxxTest::TestSuite {
                    (double (*)(double))sqrt);
     // Generate a histogram with the same mid points as the point data example
     std::generate(tof.begin(), tof.end(), Incrementer(99.5));
-    MayersSampleCorrectionStrategy mscat(createTestParameters(), tof, signal,
-                                         error);
+
+    BinEdges tofPoints(tof);
+    Histogram histo(tofPoints, Counts(signal), CountStandardDeviations(error));
+    MayersSampleCorrectionStrategy mscat(createTestParameters(), histo);
 
     // Correct it
-    mscat.apply(signal, error);
+
+    auto outHisto = mscat.getCorrectedHisto();
+
+    const auto &tofVals = outHisto.x();
+    const auto &signalVals = outHisto.y();
+    const auto &errVals = outHisto.e();
 
     // Check some values
     const double delta(1e-06);
-    TS_ASSERT_DELTA(99.5, tof.front(), delta);
-    TS_ASSERT_DELTA(199.5, tof.back(), delta);
+    TS_ASSERT_DELTA(99.5, tofVals.front(), delta);
+    TS_ASSERT_DELTA(199.5, tofVals.back(), delta);
 
-    TS_ASSERT_DELTA(0.37497317, signal.front(), delta);
-    TS_ASSERT_DELTA(0.37629282, signal.back(), delta);
+    TS_ASSERT_DELTA(0.37497317, signalVals.front(), delta);
+    TS_ASSERT_DELTA(0.37629281, signalVals.back(), delta);
```

Comment:
It was also bin edges before you changed anything, as far as I can see? So why would the values change?